### ISSUE TITLE
Unable to assign private_ip to instance (multiple)

### ISSUE THREAD

**Issue by 39mbac** [terraform-aws-modules/terraform-aws-ec2-instance#102]
Using 

```
   private_ip     = “10.10.1.10”
```

To assign static address to a single instance works great using this module.

I have a need to create multiple instances and statically set their ip’s

```
  private_ip = [“10.10.1.10”, ‘10.10.2.10”] results in the “got list should be string error”
```

Have tried setting as variable before module as in below which also does not work
```
variable “private_ip”  {
                  type = “list”
}
```
What is the proper way to provision multiple instances  and statically assign private_ip?

**Comment by antonbabenko** [comment]
You are right, currently, this module creates instances but it is not possible to give each instance a fixed private IP. Sounds like a small pull-request is missing.

**Comment by 39mbac** [comment]
Glad I am not missing something.  Not sure how to go about properly adding a pull-request ...all guidance appreciated as not having this functionality has me at a temporary standstill. Thanks!

**Comment by antonbabenko** [comment]
Fixed in [v2.2.0](https://github.com/terraform-aws-modules/terraform-aws-ec2-instance/releases/v2.2.0)

**Comment by 39mbac** [comment]
Wow...that’s service! Is this backwards compatible to TF v11 or is 12 required? As it is v.2.2.0 assume 12 only?
